heareval/tasks/pipeline.py

```diff
@@ -278,15 +287,22 @@ def split_train_test_val(self, metadata: pd.DataFrame):
 
         # Depending on whether valid and test are already present, the percentage can
         # either be the predefined percentage or 0
-        valid_perc = VALIDATION_PERCENTAGE if "valid" in splits_to_sample else 0
-        test_perc = TEST_PERCENTAGE if "test" in splits_to_sample else 0
+        validation_percentage = (
+            VALIDATION_PERCENTAGE if "valid" in splits_to_sample else 0
+        )
+        test_percentage = TEST_PERCENTAGE if "test" in splits_to_sample else 0
 
+        metadata.reset_index(drop=True, inplace=True)
```

Comment:
Why does this need to happen here?

Reply:
Not sure but it can't hurt? If upstream was mucking with it and didn't clean up after themselves